src/main/java/com/mythsoftware/rest/resource/UserResource.java

```diff
@@ -0,0 +1,40 @@
+package com.mythsoftware.rest.resource;
+
+import com.mythsoftware.rest.model.User;
+import com.mythsoftware.rest.service.UserService;
+
+import javax.inject.Inject;
+import javax.inject.Named;
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import java.util.List;
+
+/**
+ */
+@Named
+@Path("/user")
+public class UserResource {
+
+    @Inject
+    private UserService userService;
+
+    @GET
+    @Path("/getUsers")
```

Comment:
Most Restfull APIs are defined as nouns as opposed to verbs, so the path get get users would be like this:
GET /users